app/src/main/java/com/yello/presentation/onboarding/fragment/AddFriendFragment.kt

```diff
@@ -0,0 +1,34 @@
+package com.yello.presentation.onboarding.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.yello.databinding.FragmentAddfreindBinding
+import com.yello.databinding.FragmentSchoolBinding
+
+class AddFriendFragment : Fragment() {
+    private var _binding: FragmentAddfreindBinding? = null
+    private val binding: FragmentAddfreindBinding
+        get() = requireNotNull(_binding) { " _binding이 null입니다!" }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ): View {
+        _binding = FragmentAddfreindBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        // 대부분의 로직은 여기에 구현합니다.
+    }
```

Comment:
```suggestion
    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        // TODO : 대부분의 로직은 여기에 구현합니다.
    }
```
TODO를 활용해서 주석을 달면 밝은 색으로 하이라이트가 되어 보완이 필요한 부분이 어디인지 보기 쉬워집니당
참고로 프로젝트에서는 구현이 더 필요한 부분이 아니라면 가독성을 위해 주석을 최소화하는 것이 좋다고 생각합니다!